src/lib/logstore/log_store_service.cpp

```diff
@@ -116,12 +128,17 @@ void LogStoreService::stop() {
         folly::SharedMutexWritePriority::WriteHolder holder(m_logdev_map_mtx);
         m_id_logdev_map.clear();
     }
-    m_id_reserver.reset();
+}
+
+logdev_id_t LogStoreService::get_next_logdev_id() {
+    auto id = ++(m_sb->m_last_logdev_id);
+    m_sb.write();
```

Comment:
the `write()` is returning void,  curious how we ensure the write will always success?

Reply:
seems like we assert when there is any error

Reply:
Yeah, right alloc_blk is raising exception and if sync_write fails, we assert. I think if any error, then its a big issue. better assert.